web/client/components/mapcontrols/locate/LocateTool.jsx

```diff
@@ -14,15 +22,19 @@ const defaultOpt = {
         maxZoom: 18
     }
 };
-
+/**
+ * Common interface shared across multiple map types
```

Comment:
This is not an interface, this is a Component

Reply:
yeah i know, i thought to add special category and not just call it component

going to fix it